lncfg/sweeper.go

```diff
@@ -3,11 +3,14 @@ package lncfg
 import (
 	"fmt"
 	"time"
+
+	"github.com/lightningnetwork/lnd/lnwallet/chainfee"
 )
 
 //nolint:lll
 type Sweeper struct {
-	BatchWindowDuration time.Duration `long:"batchwindowduration" description:"Duration of the sweep batch window. The sweep is held back during the batch window to allow more inputs to be added and thereby lower the fee per input."`
+	BatchWindowDuration time.Duration          `long:"batchwindowduration" description:"Duration of the sweep batch window. The sweep is held back during the batch window to allow more inputs to be added and thereby lower the fee per input."`
+	MaxFeeRate          chainfee.SatPerKWeight `long:"maxfeerate" description:"Maximum fee rate in sat/kw that the sweeper is allowed to pay."`
```

Comment:
can this be in sat/vb? that way users don't need to do the conversion to sat/kw?

Reply:
yeah `sat/vb` can be nice, tho one day we may want it to be decimal or use `sat/kvb` instead in case bitcoin  goes to 1m.